logstash-input-tcp.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name          = 'logstash-input-tcp'
-  s.version         = '4.1.2'
+  s.version       = ::File.read('version').split("\n").first
```

Comment:
FWIW, Beats input also has a separate VERSION file, and being new, I haven't had an [cognitive friction](https://cogfriction.wordpress.com/about/) during the release process.